Makefile

```diff
@@ -78,12 +78,12 @@ manifests: controller-gen
 
 # Run go fmt against code
 fmt:
-	go fmt ./...
```

Comment:
what was the reason of changing `go fmt ./...`  to `find . -name '*.go' | grep -v vendor | xargs gofmt -s -w`?

Reply:
the golangci-lint checks that the code has been simplified and I reused the fmt command from other projects to apply the simplification and to ignore vendored packages (avoids linter failures on those if vendoring is ever used)